jekyll/_cci2/authentication/mfa.adoc

```diff
@@ -0,0 +1,69 @@
+---
+contentTags:
+  platform:
+  - Cloud
+---
+= Multi-factor authentication (MFA)
+:page-layout: classic-docs
+:page-liquid:
+:page-description: MFA Overview
+:icons: font
+:experimental:
+
+NOTE: MFA for CircleCI is available for users who access their accounts using email and password. Users who use social login methods to access their accounts (such as GitHub or Bitbucket), may use their login provider's MFA offering. MFA is currently optional. Starting December 1, 2025 MFA will be required for all users who log in with email and password.
+
+[#introduction]
+== Introduction
+
+Multi-factor authentication (MFA) is an additional layer of security for your CircleCI account. We strongly recommend that you set up and enable it.
+With MFA enabled, an additional verification step is required to access your account and make changes to account authentication settings, such as email or password.
+This means that even if your account password is compromised, only someone with access to the additional verification factor can
+access the account. The additional verification step takes the form of providing a One-Time Password (OTP).
+
+[#setup-mfa]
+== Setup MFA
+
+To configure MFA on your account, follow these steps:
+
+. In the link:https://app.circleci.com/home/[CircleCI web app], select your profile from the upper right corner, then select **User Settings**.
+. Select **Password & authentication** from the sidebar.
```

Comment:
```suggestion
. Select **Password** from the sidebar.
```

Reply:
Currently it is just labeled Password, is this set to change?

Reply:
Yes, we plan to change this soon: https://github.com/circleci/circleci-docs/pull/9372#discussion_r2078249070